crates/precompile/src/bls12_381.rs

```diff
@@ -2,26 +2,8 @@
 //! For more details check modules for each precompile.
 use crate::PrecompileWithAddress;
 
-cfg_if::cfg_if! {
```

Comment:
precompile/evm specific code no longer has the cfg_if code -- these were mainly being used to switch between different cryptography libraries. This now lives inside of `crypto`  folder